src/benchmarks/micro/MicroBenchmarks.csproj

```diff
@@ -56,6 +56,7 @@
     <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="$(SystemVersion)" />
     <PackageReference Include="System.Drawing.Common" Version="$(SystemVersion)" />
     <PackageReference Include="System.Formats.Cbor" Version="$(SystemVersion)" Condition="'$(TargetFramework)' == 'net5.0' or '$(TargetFramework)' == 'net6.0' or '$(TargetFramework)' == 'net7.0'" />
+    <PackageReference Include="System.Formats.Tar" Version="$(SystemVersion)" Condition="'$(TargetFramework)' == 'net7.0'" />
```

Comment:
Just checking locally, it looks like this package reference is implicitly available just by targeting .NET 7 Preview 4

Reply:
Thanks for the help offline. This was causing the restore problem. Removing the whole line fixed it.

Now I got other errors. Investigating them...

Reply:
Ok the other errors were just build failures I couldn't see before, or exceptions thrown due to incorrect setup/cleanup. All fixed now.